src/core/server/plugins/types.ts

```diff
@@ -194,6 +194,10 @@ export interface InternalPluginInfo {
    * served
    */
   readonly publicTargetDir: string;
+  /**
+   * Path to the plugin assets directory.
+   */
+  readonly publicAssetsDir: string;
```

Comment:
I don't think we want to server build artifacts from the `/assets` folder, so I separated them.